app/addons/activetasks/tests/activetasks.componentsSpec.react.jsx

```diff
@@ -70,6 +70,14 @@ define([
         window.confirm.restore && window.confirm.restore();
       });
 
+
+      it('it displays a message instead of an empty table, if there are undefined active tasks', function () {
+        spy = sinon.spy(Actions, 'setCollection');
```

Comment:
:/ i'm not really asserting that the spy is called, that's not what i'm testing for. but if i leave the spy uninitialized, the afterEach function calls spy.restore, and errors at that point.